protostar/testing/test_collector.py

```diff
@@ -279,7 +280,7 @@ def _build_test_suite_from_test_suite_info(
         test_cases = list(
             test_suite_info.filter_test_cases(
                 self._collect_test_cases(
-                    function_names=function_names,
+                    function_names=[(fn, None) for fn in function_names],
```

Comment:
Is this correct? Shouldn't this be `[fn[0] for fn in function_names]` ?

Reply:
It has to be consistent between collector versions, yup it is correct